Testscripts/Windows/STRESS-mtu-netvsc-reload.ps1

```diff
@@ -0,0 +1,77 @@
+########################################################################
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache License.
+########################################################################
+
+<#
+.Synopsis
+    MTU & netvsc reload test
+
+#>
+
+function Main {
+    param (
+        $VMName,
+        $HvServer,
+        $Ipv4,
+        $VMPort,
+        $VMUserName,
+        $VMPassword,
+        $RootDir
+    )
+    ##############################################################################
+    #
+    # Main script body
+    #
+    ##############################################################################
+    # Change working directory to root dir
+    Set-Location  $RootDir
+    LogMsg "Changed working directory to $RootDir"
+    # Start changing MTU on VM
+    $mtu_values = 1505, 2048, 4096, 8192, 16384
+    $iteration = 1
+    foreach ($i in $mtu_values) {
+        LogMsg "Changing MTU on VM to $i"
+        RunLinuxCmd -username $VMUserName -password $VMPassword -ip $Ipv4 -port $VMPort "echo 'sleep 5 && ip link set dev eth0 mtu $i &' > /home/$VMUserName/changeMTU.sh"
+        RunLinuxCmd -username $VMUserName -password $VMPassword -ip $Ipv4 -port $VMPort "bash /home/$VMUserName/changeMTU.sh > changeMTU.log 2>&1" -runAsSudo
+        Start-Sleep -s 30
+        Test-Connection -ComputerName $ipv4
+        if (-not $?) {
+            LogErr  "VM became unresponsive after changing MTU on VM to $i on iteration $iteration "
+            return "FAIL"
+        }
+        $iteration++
+    }
+    LogMsg "Successfully changed MTU for $iteration times"
+
+    # Start unloading/loading netvsc for 25 times
+    $remoteScript = "reload-netvsc.sh"
+    $stateFile = "${LogDir}\state.txt"
+    $netvsc = "echo '${VMPassword}' | sudo -S -s eval `"export HOME=``pwd``;bash ${remoteScript} > netvsc.log`""
+    RunLinuxCmd -username $VMUserName -password $VMPassword -ip $Ipv4 -port $VMPort $netvsc -runAsSudo
+    RemoteCopy -download -downloadFrom $Ipv4 -files "/home/${VMUserName}/state.txt" `
+        -downloadTo $LogDir -port $VMPort -username $VMUserName -password $VMPassword
+    RemoteCopy -download -downloadFrom $Ipv4 -files "/home/${VMUserName}/netvsc.log" `
+        -downloadTo $LogDir -port $VMPort -username $VMUserName -password $VMPassword
+    $contents = Get-Content -Path $stateFile
+    if (($contents -eq "TestAborted") -or ($contents -eq "TestFailed")) {
+        LogErr "Error: Running $remoteScript script failed on VM!"
+        return "FAIL"
+    }
+
+    Start-Sleep -s 600
+    "${vmName} IP Address after reloading hv_netvsc: ${Ipv4}"
```

Comment:
I don't see where the IP of VM is checked if it's still matching or it changed

Reply:
Added IP Verfication portion